browser/src/control/Control.UIManager.js

```diff
@@ -117,6 +119,14 @@ L.Control.UIManager = L.Control.extend({
 
 	initDarkBackgroundUI: function(activate) {
 		document.documentElement.setAttribute('data-bg-theme', activate ? 'dark' : 'light');
+		if (activate) {
+			this.map['stateChangeHandler'].setItemValue('invertbackground', 'false')
```

Comment:
I think this could be done by `map.fire` which will propagate properly the information and do the correct state update in the JSDialog, without need of manual class modification here

Reply:
`this._map.fire('commandstatechanged', {commandName : commandName, state : state});`
https://github.com/CollaboraOnline/online/blob/40d3c3d1848782d58af3d682ef9f135c500c462a/browser/src/layer/tile/CanvasTileLayer.js#L2504

it should trigger https://github.com/CollaboraOnline/online/blob/40d3c3d1848782d58af3d682ef9f135c500c462a/browser/src/control/Control.JSDialogBuilder.js#L2390